analytics-integrations/taplytics/build.gradle

```diff
@@ -0,0 +1,5 @@
+apply from: rootProject.file('gradle/integration.gradle')
+
+dependencies {
+  compile files('libs/taplytics-1.4.2.jar')
```

Comment:
Does this JAR contain all the relevant dependencies listed here - https://cloudup.com/c6GLI7ewTHR